spec/models/access_online_links_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-RSpec.describe AccessOnlineLinks do
+RSpec.describe ExternalLinks do
   describe '#access_online_links' do
```

Comment:
Probably rename this as well to `#external_links`?

Reply:
yup